nodejs/user-interface/modals/new-task.js

```diff
@@ -31,5 +31,10 @@ module.exports = (prefilledTitle) => {
           actionId: 'taskDueTime',
         }),
       ),
+      Blocks.Input({ label: 'Assign user', blockId: 'taskAssignUser'}).element(
```

Comment:
This should probably be 

- optional or default to current user
- moved above the scheduling piece